lib/collection/src/collection_manager/holders/proxy_segment.rs

```diff
@@ -830,7 +830,7 @@ mod tests {
             .unwrap();
         proxy_segment.delete_point(102, 1.into()).unwrap();
 
-        let query_vector = vec![1.0, 1.0, 1.0, 1.0];
```

Comment:
Though I don't really mind and this is in testing, `vec![]` is better because it does some compile time magic.

Reply:
Not the case anymore. Please disregard this comment. :)